src/main/java/frc/team2412/robot/util/AbsoluteSwerveModule.java

```diff
@@ -0,0 +1,56 @@
+package frc.team2412.robot.util;
+
+import com.ctre.phoenix.sensors.CANCoder;
+import com.ctre.phoenix.sensors.SensorInitializationStrategy;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+
+public class AbsoluteSwerveModule extends SwerveModule {
+
+    private CANCoder absoluteEncoder;
+    private int turnEncoderPort;
+    private double turnOffset;
+    private String canbus;
```

Comment:
`turnEncoderPort`, `turnOffset`, and `canbus` need to be initialized (and probably should be final).